### PR TITLE
Bullet list

### PR DESCRIPTION
**Pull request by Amrita42**
##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/network/index.rst [hunk 1]
```diff
@@ -4,7 +4,10 @@
 Ansible for Network Automation
 ******************************
 
-Ansible Network modules extend the benefits of simple, powerful, agentless automation to network administrators and teams. Ansible Network modules can configure your network stack, test and validate existing network state, and discover and correct network configuration drift.
+Ansible Network modules extend the benefits of simple, powerful, agentless automation to network administrators and teams. Ansible Network modules can:
+*Configure your network stack
```
Comment (samccann) [inline docs/docsite/rst/network/index.rst:8]:
```suggestion

* Configure your network stack
```

File: docs/docsite/rst/network/index.rst [hunk 2]
```diff
@@ -4,7 +4,10 @@
...
+*Test and validate existing network state
```
Comment (samccann) [inline docs/docsite/rst/network/index.rst:9]:
```suggestion
* Test and validate existing network state
```

File: docs/docsite/rst/network/index.rst [hunk 3]
```diff
@@ -4,7 +4,10 @@
...
+*Discover and correct network configuration drift
```
Comment (samccann) [inline docs/docsite/rst/network/index.rst:10]:
```suggestion
* Discover and correct network configuration drift
```

**Comment by acozine** [comment]
I like this idea, however, placing a bulleted list in the first paragraph turns the beginning of the paragraph into some kind of a header, with odd styling:

Screen Shot 2019-11-12 at 1 17 07 PM

I've tried a few things to make this work, but so far nothing has restored that section to normal styling. Thanks so much @Amrita42 for trying this out, but we cannot implement this change at this time.

**Comment by Amrita42** [comment]
@acozine Sorry for the delay in response. Welcome and thanks for trying to make it work,maybe next time when styling works out.

**Pull request closed without merging**